tests/types/ember-concurrency-test.ts

```diff
@@ -925,99 +925,99 @@ module('unit tests', () => {
     let O = EmberObject.extend({
       tp,
 
-      foo() {
-        let t = this.get('tp');
+      foo(this: EmberObject) {
+        let t = this.get('tp') as Task<string, [boolean, number?]>;
```

Comment:
With preview types, `get` generally returns `unknown`. I used casts to get around that. It's not pretty.